client/tests/acceptance/user-can-click-package-edit-test.js

```diff
@@ -63,15 +63,6 @@ module('Acceptance | user can click package edit', function(hooks) {
 
     await visit('/packages/2/edit');
 
-    // ! We remove the pregenerated applicant here to disable the "save" button.
```

Comment:
@godfreyyeung I removed this entire comment because we no longer have to deal with this, but let me know if there are parts of this that you think should still be present.